Tests/UnitTests/ReaderTests/Data/ImageReaderSimpleRegression_map.txt

```diff
@@ -0,0 +1,4 @@
+images\11.png	1	1
```

Comment:
I can do that. I have to admit though, that I messed up something in my python environment and cannot run the E2E tests anymore on my machine. I will have to fix that first, so it may take some time O:-)